test/azure-pipelines.yml

```diff
@@ -0,0 +1,216 @@
+      cat dashboard.cmake
+    displayName: 'Configure CTest script'
+
+  - script: |
+      $(compilerInitialization)
+      ctest -j 2 -V -S dashboard.cmake
+    displayName: 'Build and test'
+
+  - script: |
+      sudo pip3 install --upgrade pip
+      sudo pip3 install --upgrade setuptools
+      sudo pip3 install scikit-ci-addons
+      ci_addons ctest_junit_formatter $(Agent.BuildDirectory)/build > $(Agent.BuildDirectory)/JUnitTestResults.xml
+    condition: succeededOrFailed()
+    displayName: 'Format CTest output in JUnit format'
+
+  - task: PublishTestResults@2
+    inputs:
+      testResultsFiles: "$(Agent.BuildDirectory)/JUnitTestResults.xml"
+      testRunTitle: 'CTest $(Agent.OS)'
+    condition: succeededOrFailed()
+    displayName: 'Publish test results'
+
+
+- job: 'PackageLinux'
+  timeoutInMinutes: 0
+  cancelTimeoutInMinutes: 300
+  displayName: "Build Linux Python packages"
+  pool:
+    vmImage: 'Ubuntu-16.04'
+
+  steps:
+  - script: |
+      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
+      chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
+    displayName: 'Fetch build script'
+
+  - script: |
+      export ITK_PACKAGE_VERSION=$(ITKGitTag)
+      ./dockcross-manylinux-download-cache-and-build-module-wheels.sh
+    displayName: 'Build Python packages'
+
+  - task: PublishPipelineArtifact@0
+    inputs:
+      artifactName: 'LinuxWheels'
+      targetPath: './dist'
+
+
+- job: 'PackageMacOS'
+  displayName: "Build macOS Python packages"
+  timeoutInMinutes: 0
+  cancelTimeoutInMinutes: 300
+  pool:
+    vmImage: 'macos-10.13'
+
+  steps:
+  - script: |
+      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/macpython-download-cache-and-build-module-wheels.sh -O
+      chmod u+x macpython-download-cache-and-build-module-wheels.sh
+    displayName: 'Fetch build script'
+
+  - script: |
+      export ITK_PACKAGE_VERSION=$(ITKGitTag)
+      ./macpython-download-cache-and-build-module-wheels.sh
+    displayName: 'Build Python packages'
+
+  - task: PublishPipelineArtifact@0
+    inputs:
+      artifactName: 'MacOSWheels'
+      targetPath: './dist'
+
+
+- job: 'PackageWindows'
+  displayName: "Build Windows Python packages"
+  timeoutInMinutes: 0
+  cancelTimeoutInMinutes: 300
+  pool:
+    vmImage: 'vs2017-win2016'
+
+  steps:
+  - script: |
+      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/windows-download-cache-and-build-module-wheels.ps1 -O
+    displayName: 'Fetch build script'
+
+  - script: |
+      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+      set ITK_PACKAGE_VERSION=$(ITKGitTag)
+      set CC=cl.exe
+      set CXX=cl.exe
+      powershell.exe -file .\windows-download-cache-and-build-module-wheels.ps1
```

Comment:
Here and above we want to use the ITKVtkGlue specific scripts:

  https://github.com/InsightSoftwareConsortium/ITKVtkGlue/blob/90084ac653f03d51e4606a7f59bbe0777febb426/appveyor.yml#L10-L11

Reply:
yes, I had messed that up when rebasing. It should be back to what it was before my rebase, and it should work.